aleph/views/stream_api.py

```diff
@@ -53,4 +64,21 @@ def entities(collection_id=None):
         schemata=schemata,
         includes=includes,
     )
-    return stream_ijson(entities)
+
+    STREAMS.inc()
+
+    def generator(entities):
+        count = 0
+
+        for entity in entities:
+            count += 1
+
+            if count == 1000:
+                STREAMED_ENTITIES.inc(count)
+                count = 0
+
+            yield entity
+
+        STREAMED_ENTITIES.inc(count)
```

Comment:
I initially thought that increasing the counter on every iteration might be a bad idea -- that’s why I incremented it only every 1k entities. But thinking about it again, this might be a premature optimization, I should double-check that.